### ISSUE TITLE
incorrect conversion of some bibtex types?

### ISSUE THREAD

**Issue by samwalls** [citation-js/citation-js#126]
Hi there,

I'm trying out the [demo site](https://citation.js.org/demo/) and if I input the following into the BibTex box:
```
@artwork{ArtTest,
  author = {Test},
  year = {1800},
  title = {My Artwork},
}
```
or
```
@misc{Test,
  author = {Test},
  year = {1800},
  title = {My Artwork},
}
```

When "bibtex" or "bib.txt" is selected as the output type, I see that both are converted to the `@book` type. Is this expected behaviour? How can I preserve the type? Either this is an issue with the library, or something misconfigured about the demo.

**Comment by larsgw** [comment]
The demo might run an outdated version, I will check that tomorrow. However, CSL 1.0.1 (the format Citation.js uses internally) does not have a generic document type to represent `@artwork` or `@misc`. This changes in 1.0.2 but that's not completely released yet, or supported by downstream projects. For that reason, "book" has generally been recommended as default type (I think). However there might be types that are better fallbacks for specific types like `@artwork`>

**Comment by samwalls** [comment]
For `@artwork` I believe it's meant to be functionally equivalent to `@misc`. I'm not quite sure what defines that, but I think I remember seeing it in the biblatex manual somewhere.

I tried this out on citation-js 0.5.1 earlier today just to be sure, and I see the same behaviour as you mentioned.

I'd recommend support for `misc` at a minimum, if something like this is in the pipeline, because it's the most common suggestion I see for anything that's atypical.

**Comment by larsgw** [comment]
Yes, the biblatex manual lists `@artwork` under Non-standard Types (2.1.3), adding that they do not have their own drivers, and use the one of `@misc` instead.

 > I'd recommend support for `misc` at a minimum, if something like this is in the pipeline, because it's the most common suggestion I see for anything that's atypical.

It's more or less in the pipeline, but I either have to wait on CSL 1.0.2 to release, or replace CSL 1.0.1 with something else which is going to take time, both planning and actually carrying it out. If you happen to have any recommendations of general-purpose but relatively comprehensive bibliographical ontologies, I would be happy to hear about them (#127).

**Comment by larsgw** [comment]
Fixed in 6c68aff80d3e04e132c722ef508d0a45fad40a29
